<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleWindow</class>
 <widget class="QMainWindow" name="ScheduleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Schedule</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidgetSchedule">
        <property name="columnCount">
         <number>3</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">Action</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Params</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string notr="true">Condition</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="title">
         <string>New schedule record:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Action</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxAction"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Profile</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxProfiles"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Condition(e.g .*15:37 or .*Mon.*12:40 )</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCondition">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Regular Expressions syntax: &lt;a href=&quot;https://doc.qt.io/qt-4.8/qregexp.html#introduction&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;https://doc.qt.io/qt-4.8/qregexp.html#introduction&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Current date and time:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelCurrentDateTime">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Add new record</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonDelete">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Delete selected record</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
